diff --git a/dist/compiled/react-server-dom-turbopack/cjs/react-server-dom-turbopack-client.browser.development.js b/dist/compiled/react-server-dom-turbopack/cjs/react-server-dom-turbopack-client.browser.development.js
index e7612dd1c4522a9ea8dad033b07567b08bcd1754..eb5f6beef9e435824035980277ef9ae542127000 100644
--- a/dist/compiled/react-server-dom-turbopack/cjs/react-server-dom-turbopack-client.browser.development.js
+++ b/dist/compiled/react-server-dom-turbopack/cjs/react-server-dom-turbopack-client.browser.development.js
@@ -3809,7 +3809,10 @@
                       startTime$jscomp$2 = startTime$jscomp$1,
                       childrenEndTime$jscomp$1 = childrenEndTime$jscomp$0,
                       error = root.reason;
-                    if (supportsUserTiming) {
+                    if (
+                      supportsUserTiming &&
+                      0 <= childrenEndTime$jscomp$1
+                    ) {
                       var env = componentInfo$jscomp$2.env,
                         name = componentInfo$jscomp$2.name,
                         entryName$jscomp$0 =
@@ -4072,7 +4075,10 @@
                     trackIdx$jscomp$4 = trackIdx$jscomp$6,
                     startTime$jscomp$5 = time,
                     childrenEndTime$jscomp$3 = childrenEndTime;
-                  if (supportsUserTiming) {
+                  if (
+                    supportsUserTiming &&
+                    0 <= childrenEndTime$jscomp$3
+                  ) {
                     var env$jscomp$2 = componentInfo$jscomp$4.env,
                       name$jscomp$1 = componentInfo$jscomp$4.name,
                       entryName$jscomp$2 =
diff --git a/dist/compiled/react-server-dom-turbopack-experimental/cjs/react-server-dom-turbopack-client.browser.development.js b/dist/compiled/react-server-dom-turbopack-experimental/cjs/react-server-dom-turbopack-client.browser.development.js
index e7612dd1c4522a9ea8dad033b07567b08bcd1754..eb5f6beef9e435824035980277ef9ae542127000 100644
--- a/dist/compiled/react-server-dom-turbopack-experimental/cjs/react-server-dom-turbopack-client.browser.development.js
+++ b/dist/compiled/react-server-dom-turbopack-experimental/cjs/react-server-dom-turbopack-client.browser.development.js
@@ -3809,7 +3809,10 @@
                       startTime$jscomp$2 = startTime$jscomp$1,
                       childrenEndTime$jscomp$1 = childrenEndTime$jscomp$0,
                       error = root.reason;
-                    if (supportsUserTiming) {
+                    if (
+                      supportsUserTiming &&
+                      0 <= childrenEndTime$jscomp$1
+                    ) {
                       var env = componentInfo$jscomp$2.env,
                         name = componentInfo$jscomp$2.name,
                         entryName$jscomp$0 =
@@ -4072,7 +4075,10 @@
                     trackIdx$jscomp$4 = trackIdx$jscomp$6,
                     startTime$jscomp$5 = time,
                     childrenEndTime$jscomp$3 = childrenEndTime;
-                  if (supportsUserTiming) {
+                  if (
+                    supportsUserTiming &&
+                    0 <= childrenEndTime$jscomp$3
+                  ) {
                     var env$jscomp$2 = componentInfo$jscomp$4.env,
                       name$jscomp$1 = componentInfo$jscomp$4.name,
                       entryName$jscomp$2 =
diff --git a/dist/compiled/react-server-dom-webpack/cjs/react-server-dom-webpack-client.browser.development.js b/dist/compiled/react-server-dom-webpack/cjs/react-server-dom-webpack-client.browser.development.js
index d80dca55bbf43172401d26348176ae57d7d69566..b5a557f34860e643be4fef50f344b08ac718ca93 100644
--- a/dist/compiled/react-server-dom-webpack/cjs/react-server-dom-webpack-client.browser.development.js
+++ b/dist/compiled/react-server-dom-webpack/cjs/react-server-dom-webpack-client.browser.development.js
@@ -3818,7 +3818,10 @@
                       startTime$jscomp$2 = startTime$jscomp$1,
                       childrenEndTime$jscomp$1 = childrenEndTime$jscomp$0,
                       error = root.reason;
-                    if (supportsUserTiming) {
+                    if (
+                      supportsUserTiming &&
+                      0 <= childrenEndTime$jscomp$1
+                    ) {
                       var env = componentInfo$jscomp$2.env,
                         name = componentInfo$jscomp$2.name,
                         entryName$jscomp$0 =
@@ -4081,7 +4084,10 @@
                     trackIdx$jscomp$4 = trackIdx$jscomp$6,
                     startTime$jscomp$5 = time,
                     childrenEndTime$jscomp$3 = childrenEndTime;
-                  if (supportsUserTiming) {
+                  if (
+                    supportsUserTiming &&
+                    0 <= childrenEndTime$jscomp$3
+                  ) {
                     var env$jscomp$2 = componentInfo$jscomp$4.env,
                       name$jscomp$1 = componentInfo$jscomp$4.name,
                       entryName$jscomp$2 =
diff --git a/dist/compiled/react-server-dom-webpack-experimental/cjs/react-server-dom-webpack-client.browser.development.js b/dist/compiled/react-server-dom-webpack-experimental/cjs/react-server-dom-webpack-client.browser.development.js
index d80dca55bbf43172401d26348176ae57d7d69566..b5a557f34860e643be4fef50f344b08ac718ca93 100644
--- a/dist/compiled/react-server-dom-webpack-experimental/cjs/react-server-dom-webpack-client.browser.development.js
+++ b/dist/compiled/react-server-dom-webpack-experimental/cjs/react-server-dom-webpack-client.browser.development.js
@@ -3818,7 +3818,10 @@
                       startTime$jscomp$2 = startTime$jscomp$1,
                       childrenEndTime$jscomp$1 = childrenEndTime$jscomp$0,
                       error = root.reason;
-                    if (supportsUserTiming) {
+                    if (
+                      supportsUserTiming &&
+                      0 <= childrenEndTime$jscomp$1
+                    ) {
                       var env = componentInfo$jscomp$2.env,
                         name = componentInfo$jscomp$2.name,
                         entryName$jscomp$0 =
@@ -4081,7 +4084,10 @@
                     trackIdx$jscomp$4 = trackIdx$jscomp$6,
                     startTime$jscomp$5 = time,
                     childrenEndTime$jscomp$3 = childrenEndTime;
-                  if (supportsUserTiming) {
+                  if (
+                    supportsUserTiming &&
+                    0 <= childrenEndTime$jscomp$3
+                  ) {
                     var env$jscomp$2 = componentInfo$jscomp$4.env,
                       name$jscomp$1 = componentInfo$jscomp$4.name,
                       entryName$jscomp$2 =
